### DISCUSSION TITLE
Suggestion: Pin topic #1 ('welcome to meshtastic discussions')

### DISCUSSION THREAD

**Discussion by easytarget** [meshtastic/Discussions#26] [General]
This seems to generally be best practice; and gives a place for new users to land.

**Comment by rcarteraz** [comment]
It's pinned to the announcements and shows as a banner at the top of Discussions. 
<img width="1275" alt="Screenshot 2024-10-30 at 1 57 14 PM" src="https://github.com/user-attachments/assets/fb843b0c-5ae7-4fc8-92f4-ac9c7b2ed4ac">

**Comment by easytarget** [comment]
Ah!, yes, my bad.
I was assuming GH discussions would also put at the top of the main list on the discussions page itself too. But I guess that's redundant, sorry for the interruption..
